Makefile

```diff
@@ -1,6 +1,7 @@
 all:
 	@echo "Only tests are available: make test"
 
+.PHONY: test
```

Comment:
It is relevant to the commit, where you added luatest based tests within the `test/` directory. Not here.

Reply:
Removed from the patch.